### ISSUE TITLE
loopback plugin should pass prevResult through

### ISSUE THREAD

**Issue by dcbw** [containernetworking/plugins#376]
Loopback always returns an empty result, so if it's part of a chain it will overwrite any result from a previous plugin. It should just pass the prevResult through unchanged for configs >= 0.3.0.

(workaround is to always put loopback first in a chain)

see https://github.com/containernetworking/plugins/pull/374

**Comment by mars1024** [comment]
Yes, there are two kinds of plugins, one is main plugin for generating result, the other is chained plugin for reading previous result and passing it (in some case, pass after updating previous result). Bridge, macvlan, host-device... are all main plugins, there should be only one in the whole plugin chain. Portmap, bandwidth... are all chained plugin, they must follow by main plugins.

But loopback is a special plugin, it looks like a main plugin because it doesn't need to follow by main plugins, but it should be a special chained plugin which can be at any position of a plugin chain.
